noxfile.py

```diff
@@ -33,7 +33,14 @@ def default(session):
     """
     # Install all test dependencies, then install local packages in-place.
     session.install(
-        "mock", "pytest", "google-cloud-testutils", "pytest-cov", "freezegun"
+        "mock",
+        "pytest",
+        "google-cloud-testutils",
+        "pytest-cov",
+        "freezegun",
+        "opentelemetry-api",
```

Comment:
Ideally all of these extra dependencies would be included in the `[all]` extras used below.

Reply:
I am not entirely sure where `[all]` is defined. I am also not completely familiar with nox.

Reply:
Now that you've added `opentelemetry` to https://github.com/googleapis/python-bigquery/blob/bae233770463db697df63dc66f9a6f54bd10d437/setup.py#L38 these dependencies are all included in `[all]`. It's defined here:

https://github.com/googleapis/python-bigquery/blob/bae233770463db697df63dc66f9a6f54bd10d437/setup.py#L65-L75